instrumentation-security/spymemcached-2.12.0/src/test/java/com/nr/agent/security/instrumentation/memcached/test/MemcachedTest.java

```diff
@@ -0,0 +1,199 @@
+package com.nr.agent.security.instrumentation.memcached.test;
+
+import com.github.mwarc.embeddedmemcached.JMemcachedServer;
+import com.newrelic.agent.security.introspec.InstrumentationTestConfig;
+import com.newrelic.agent.security.introspec.SecurityInstrumentationTestRunner;
+import com.newrelic.agent.security.introspec.SecurityIntrospector;
+import com.newrelic.api.agent.security.schema.AbstractOperation;
+import com.newrelic.api.agent.security.schema.VulnerabilityCaseType;
+import com.newrelic.api.agent.security.schema.operation.MemcachedOperation;
+import net.spy.memcached.MemcachedClient;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.MethodSorters;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.util.List;
+
+@RunWith(SecurityInstrumentationTestRunner.class)
+@InstrumentationTestConfig(includePrefixes = { "net.spy.memcached" })
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class MemcachedTest {
+
+    private static MemcachedClient memcachedClient;
+    private final String key = "someKey";
+    private final String value = "value";
+    private final int expirationInSeconds = 1800;
+    private final long casID = 1;
+    private static JMemcachedServer server;
+
+    @BeforeClass
+    public static void setup() throws IOException {
+        server = new JMemcachedServer();
+        server.start("127.0.0.1", 11214);
+        memcachedClient = new MemcachedClient(new InetSocketAddress("127.0.0.1", 11214));
+    }
+
+    @AfterClass
+    public static void stop() {
+        memcachedClient.flush();
+        memcachedClient.shutdown();
+        server.clean();
+    }
+    @Test
+    public void setTest() {
```

Comment:
follow the naming convention that we have been using for the method name `testSet()`

Reply:
Updated in this [commit](https://github.com/newrelic/csec-java-agent/pull/111/commits/cc2abafee8e14573711eb65bb718458d1efc55ef)